tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_SimpleWenoImpl.cpp

```diff
@@ -526,21 +558,27 @@ void test_simple_weno_3d(const std::unordered_set<Direction<3>>&
                            neighbor_modified_vars, custom_approx);
 }
 
+void test_simple_weno_3d() noexcept {
+  INFO("Test simple_weno_impl in 3D");
+  const auto gl = Spectral::Quadrature::GaussLobatto;
+  const auto gauss = Spectral::Quadrature::Gauss;
+
+  test_simple_weno_3d_impl(gl);
+  test_simple_weno_3d_impl(gauss);
+
+  // Test with particular boundaries labeled as external
+  test_simple_weno_3d_impl(gl, {{Direction<3>::lower_zeta()}});
```

Comment:
maybe either indicate that gauss points don't work in this case or test gauss points too?